rss-client/src/app/interfaces/user.ts

```diff
@@ -2,9 +2,8 @@ export interface User {
   userId: number,
   email: string,
   password: string,
-  profilePic: string,
+  profilePic: any,
```

Comment:
Shouldn't be a problem, but we'd like to know why this change is necessary.

Reply:
It's how it looked before my giant pull request. Just wanted to change it back.